docs/source/upgrade_to_newest_version.md

```diff
@@ -1,10 +1,10 @@
-# Considerations for getting to v2.x
+# Considerations for getting to v2.0
```

Comment:
No change needed here, the original text makes sense.

In fact, the changes proposed in this file changes the whole meaning. I'm going to go ahead and close the PR since this type of "replace all" change is not beneficial and can actually change the meaning from the original authors.